### DISCUSSION TITLE
Tailwind Colors - supported out of the box?

### DISCUSSION THREAD

**Discussion by hydrohiester** [tremorlabs/tremor-npm#1055] [General]
Are tailwind colors supposed to just work out of the box? Like if I have a DonutChart and I specify `colors={["gray-500"]}`, etc.

Is that supposed to "just work"? (It's not working for me on an existing NextJS app with tailwind css.

**Comment by alanfrancis442** [comment]
can you share the code

**Reply by severinlandolt** [reply]
Hi! You need to use the specified set of colors from here: https://npm.tremor.so/docs/layout/color-palette

This is due to how we generate the colors under the hood. However, it's absolutely possible to use custom colors, check out this page: https://npm.tremor.so/docs/visualizations/donut-chart#usage-example-with-a-custom-colors
